terraform/scripts/create_bootstrap_bundle.sh

```diff
@@ -132,53 +131,55 @@ done
 artifact_dir=${sandbox_dir}/hab/cache/artifacts
 log "Creating TAR for all artifacts"
 
-sup_artifact=$(echo ${artifact_dir}/core-hab-sup-*)
+sup_artifact=$(echo "${artifact_dir}"/core-hab-sup-*)
 archive_name=${this_bootstrap_bundle}.tar
 log "Generating archive: ${archive_name}"
 
 tar --create \
        --verbose \
-       --file=${archive_name} \
-       --directory=${sandbox_dir}/hab/cache \
+       --file="${archive_name}" \
+       --directory="${sandbox_dir}"/hab/cache \
        artifacts >&2
 
 # We'll need a hab binary to bootstrap ourselves; let's take the one
 # we just downloaded, shall we?
-hab_pkg_dir=$(echo ${sandbox_dir}/hab/pkgs/core/hab/${hab_version}/*)
+hab_pkg_dir=$(echo "${sandbox_dir}"/hab/pkgs/core/hab/"${hab_version}"/*)
 tar --append \
        --verbose \
-       --file=${archive_name} \
-       --directory=${hab_pkg_dir} \
+       --file="${archive_name}" \
+       --directory="${hab_pkg_dir}" \
        bin >&2
 
 # We're also going to need the public origin key(s)!
 tar --append \
        --verbose \
-       --file=${archive_name} \
-       --directory=${sandbox_dir}/hab/cache \
+       --file="${archive_name}" \
+       --directory="${sandbox_dir}"/hab/cache \
        keys >&2
 
 ########################################################################
 # Upload to S3
 
-checksum=$(sha256sum ${archive_name} | awk '{print $1}')
+checksum=$(sha256sum "${archive_name}" | awk '{print $1}')
 
 # Encapsulate the fact that we want our uploaded files to be publicly
 # accessible.
 s3_cp() {
-  aws s3 cp --acl=public-read ${1} ${2} >&2
+  aws s3 cp --acl=public-read "${1}" "${2}" >&2
 }
 
-s3_cp ${archive_name} s3://${s3_bucket}
+s3_cp "${archive_name}" s3://${s3_bucket}
 
 manifest_file=${this_bootstrap_bundle}_manifest.txt
-echo ${archive_name} > ${manifest_file}
-echo ${checksum} >> ${manifest_file}
-echo >> ${manifest_file}
-tar --list --file ${archive_name} | sort >> ${manifest_file}
-
-s3_cp ${manifest_file} s3://${s3_bucket}
-s3_cp s3://${s3_bucket}/${manifest_file} s3://${s3_bucket}/LATEST
-
-rm -rdf $sandbox_dir
-rm -Rdf $archive_name
+{
+  echo "${archive_name}"
+  echo "${checksum}"
+  echo
+  tar --list --file "${archive_name}" | sort
+} > "${manifest_file}"
```

Comment:
See https://github.com/koalaman/shellcheck/wiki/SC2129#consider-using--cmd1-cmd2---file-instead-of-individual-redirects